changelogs/fragments/850-proxmox_kvm-remove_hard_coded_defaults.yml

```
@@ -0,0 +1,3 @@
+breaking_changes:
+- proxmox - remove hard-coded defaults which are different from proxmox or could be changed later by proxmox. the defaults removed were disks=3, cores=1, cpus=1, cpuunits=1, memory=512, onboot=False, swap=0 and these rely on proxmox defaults. (https://github.com/ansible-collections/community.general/pull/850).
```

```suggestion
- proxmox - remove hard-coded defaults which are different from proxmox or could be changed later by proxmox. The defaults removed were ``disks=3``, ``cores=1``, ``cpus=1``, ``cpuunits=1``, ``memory=512``, ``onboot=false``, ``swap=0``. Not specified values are set to their proxmox defaults by proxmox itself (https://github.com/ansible-collections/community.general/pull/850).
```
Similar for the other.

---

Fixed in https://github.com/ansible-collections/community.general/pull/850/commits/d95e7dde22cda3ab8ad759c94abbcc191cf1faa8 and https://github.com/ansible-collections/community.general/pull/850/commits/20ea2451d2cf3a76b7ef8600723af314534930cd.